running.md

```diff
@@ -0,0 +1,144 @@
+## Debian Package Install
+
+`sogs-standalone` package runs sogs bare and it's the fastest and simplest setup (recommended).
+
+`sogs-proxied` package runs sogs behind an nginx reverse proxy and allows for more advance flexible 
+configuration, for experienced admins only.
+
+To install the debian package for sogs (`sogs-standalone` or `sogs-proxied` ) do the following:
+
+```
+sudo curl -so /etc/apt/trusted.gpg.d/oxen.gpg https://deb.oxen.io/pub.gpg
+echo "deb https://deb.oxen.io $(lsb_release -sc) main" | sudo tee /etc/apt/sources.list.d/oxen.list
+sudo apt update
+sudo apt install sogs-standalone
+```
+
+It will prompt for server name on install.
+
+Add a room to the server:
+
+```bash
+sogs --add-room room_name_here
+```
+
+Add yourself as an admin:
+
+
+```bash
+sogs --rooms room_name_here --add-moderators SESSIONID --admin
+```
+
+
+Visit your server via http in a browser to grab the room links via the web viewer.
+
+
+
+## Manual Installation Instructions
+
+### Step 0: Do Not Run PySOGS as root
+
+Do not run pysogs as root.
+
+### Step 1: Clone the PySOGS repo:
+
+
+```bash
+git clone https://github.com/oxen-io/session-pysogs -b stable pysogs
+cd pysogs
+```
+
+### Step 2: Grab dependencies:
+
+If you are on a debian based system these can be fetched from the `deb.oxen.io` repo, to add that
+repo and install dependancies execute the following
+
+```bash
+sudo curl -so /etc/apt/trusted.gpg.d/oxen.gpg https://deb.oxen.io/pub.gpg
+echo "deb https://deb.oxen.io $(lsb_release -sc) main" | sudo tee /etc/apt/sources.list.d/oxen.list
+sudo apt update
+sudo apt install python3-{oxenmq,oxenc,pyonionreq,coloredlogs,uwsgidecorators,flask,cryptography,nacl,pil,protobuf,openssl,qrencode} uwsgi-plugin-python3
+```
+
+### Step 3: Adjust configuration
+
+Make copy of uwsgi.ini file
+
+```bash
+cp contrib/uwsgi-sogs-direct.ini uwsgi-sogs.ini
```

Comment:
```suggestion
cp contrib/uwsgi-sogs-standalone.ini uwsgi-sogs.ini
```
But you may also want to direct users to `uwsgi-sogs-proxied.ini` if they want to use a proxied configuration.